### PR TITLE
Add to comment instead of exchanging

### LINKED ISSUE #448: Trailblazer cancel overwrites existing comments
When running `trailblazer cancel ANALYSIS_ID` - It overwrites any comment and writes `Analysis cancelled manually by user: USER!`

**Solution:**
- Instead of overwriting a comment, append to a comment to keep any previous comments added to that field.

### PR DESCRIPTION
**Pull request by islean**
## Description

Closes #448.

### Fixed

- Cancelled analyses do not get their comments overridden.

### How to prepare for test
- [ ] ssh to hasta (depending on type of change)
- [ ] activate stage: `us`
- [ ] request trailblazer-stage on hasta: `paxa`
- [ ] install on stage:
    ```Shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_trailblazer -t trailblazer -b add-to-comment -a
    ```
- [ ] ssh to clinical-db (depending on type of change)
- [ ] install on stage:
`bash /home/proj/production/servers/resources/clinical-db.scilifelab.se/update-trailblazer-ui-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [ ] login to ...
- [ ] do ...

### Expected test outcome
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: trailblazer/store/crud/update.py [hunk 1]
```diff
@@ -179,10 +179,11 @@ def cancel_ongoing_analysis(
         LOG.info(f"Case {analysis.case_id} - Analysis {analysis.id}: cancelled successfully!")
         self.update_run_status(analysis_id=analysis_id, analysis_host=analysis_host)
         analysis.status = TrailblazerStatus.CANCELLED
-        analysis.comment = (
+        new_comment: str = (
             f"Analysis cancelled manually by user:"
             f" {(self.get_user(email=email).name if self.get_user(email=email) else (email or 'Unknown'))}!"
```
Comment (henrikstranneheim) [inline trailblazer/store/crud/update.py:184]: Could adding who be done in a function?

**Review by henrikstranneheim — APPROVED** [review]
Great! 👍

File: trailblazer/store/crud/update.py (hunk 1 above)
Comment (islean) [reply trailblazer/store/crud/update.py:184]: Refactored another method slightly so it could be reused for this purpose now.

**Comment by islean** [comment]
Tested in stage:

image

**Comment by islean** [comment]
Deployed to hasta and cg-services-prod

```
Log deploy... done.
trailblazer, version 21.5.16
remote: Enumerating objects: 142, done.
remote: Counting objects: 100% (142/142), done.
remote: Compressing objects: 100% (80/80), done.
remote: Total 142 (delta 79), reused 109 (delta 62), pack-reused 0
Receiving objects: 100% (142/142), 32.88 KiB | 0 bytes/s, done.
Resolving deltas: 100% (79/79), completed with 13 local objects.
From github.com:Clinical-Genomics/trailblazer
   ae5a450..ea537c5  master     -> origin/master
 * [new branch]      fix-microsalt-bug -> origin/fix-microsalt-bug
 * [new branch]      test-fix   -> origin/test-fix
 * [new branch]      update-deploy-instructions -> origin/update-deploy-instructions
 * [new tag]         21.5.16    -> 21.5.16
From github.com:Clinical-Genomics/trailblazer
 * [new tag]         21.5.10    -> 21.5.10
 * [new tag]         21.5.11    -> 21.5.11
 * [new tag]         21.5.12    -> 21.5.12
 * [new tag]         21.5.13    -> 21.5.13
 * [new tag]         21.5.14    -> 21.5.14
 * [new tag]         21.5.15    -> 21.5.15
 * [new tag]         21.5.8     -> 21.5.8
 * [new tag]         21.5.9     -> 21.5.9
Already on 'master'
Your branch is behind 'origin/master' by 18 commits, and can be fast-forwarded.
  (use "git pull" to update your local branch)
Updating ae5a450..ea537c5
Fast-forward
 .bumpversion.cfg                                          |  2 +-
 .github/workflows/build_cli_image.yml                     | 35 -----------------------------------
 .github/workflows/build_dockerfile_on_push.yml            | 15 +++++++++------
 .github/workflows/build_dockerfile_on_release.yml         | 15 +++++++++------
 setup.py                                                  |  2 +-
 tests/services/test_analysis_service.py                   | 11 +++++++----
 tests/store/crud/test_read.py                             |  4 ++--
 tests/store/crud/test_update.py                           |  6 +++---
 trailblazer/exceptions.py                                 | 10 ++++++++++
 trailblazer/server/api.py                                 |  2 +-
 trailblazer/services/analysis_service/analysis_service.py | 28 +++++++++++++++++++++-------
 trailblazer/services/job_service/job_service.py           |  6 ++++++
 trailblazer/services/job_service/utils.py                 | 24 ++++++++++++++++--------
 trailblazer/store/crud/update.py                          |  9 +++++++--
 14 files changed, 93 insertions(+), 76 deletions(-)
 delete mode 100644 .github/workflows/build_cli_image.yml
 create mode 100644 trailblazer/exceptions.py
/home/proj/stage/bin/git/trailblazer
INFO  [alembic.runtime.migration] Context impl MySQLImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
[isak.ohlsson@hasta:trailblazer] [P_base] master ±  

```

**Pull request merged**